src/System.ServiceModel.Primitives/ref/project.lock.json

```diff
@@ -2209,21 +2209,21 @@
         "lib/net46/System.Security.Cryptography.Primitives.dll",
         "lib/xamarinios10/_._",
         "lib/xamarinmac20/_._",
-        "ref/dotnet5.4/System.Security.Cryptography.Primitives.dll",
+        "ref/dotnet5.1/System.Security.Cryptography.Primitives.dll",
```

Comment:
On further testing, looks like the RestorePackages target still updates this. Also moving the generation down to 5.1 means that the Crypto libraries are now available to more platforms. It doesn't mean we're now using an older version.